bootstrap/examples/kubeflow_toolbox.yaml

```diff
@@ -0,0 +1,66 @@
+# Deploy kubeflow to namespace "kubeflow"
+
+# TODO(inc0): Create rbac role bindings
+---
+apiVersion: v1
+kind: Namespace
```

Comment:
I don't think Namespace should be inside the YAML file. This is too dangerous IMO.
If you do kubeflow delete -f kubeflow_toolbox.yaml I don't think we should automatically delete the namespace and everything in the namespace. I only think we should delete the statefulset.

I think users should manually create the namespace and manually delete it if they want to tear all resources down.

Reply:
well point is to be able to quickly pave whole thing. This definition doesn't allow parameters so it will be either kubeflow namespace or default namespace.

Reply:
I think it's beneficial to keep it until we make it fully parametrized (for example via bootstrapper).

Reply:
You can still deploy it into namespace kubeflow and just tell the users to create the namespace kubeflow manually. 

```
kubectl create namespace kubeflow
kubectl create -f mynamespace
```
Consider the case where the user had a preexisting namespace kubeflow in their cluster and now they do kubectl delete -f yourmanifest   

The idea that this could delete a preexisting namespace in my cluster is not all expected.